packages/cozy-doctypes/src/banking/label-slugs.js

```diff
@@ -1,33 +1,105 @@
 module.exports = {
+  'American Express': 'americanexpress',
+  'Arkéa Banque Privée': 'arkea',
   'AXA Banque': 'axabanque102',
-  '/Banque Populaire.*/': 'banquepopulaire',
+  'Floa Bank': 'banquecasino',
+  'Banque Populaire': 'banquepopulaire',
+  Milleis: 'barclays136',
   BforBank: 'bforbank97',
   'BNP Paribas': 'bnpparibas82',
-  BNPP: 'bnpparibas82',
-  '/Boursorama.*/': 'boursorama83',
-  casden: 'casden173',
-  '/Hello bank!.*/': 'hellobank145',
-  Bred: 'bred',
-  CA: 'caatlantica3',
+  Bolden: 'bolden',
+  Boursorama: 'boursorama83',
+  BRED: 'bred',
+  "Caisse d'Épargne Particuliers": 'caissedepargne1',
```

Comment:
Since BI, we have only 3 institutionLabels associated to cic45 and they are listed here, for example. And I see that institutionLabel is updated when an account is updated